cfme/containers/image.py

```diff
@@ -8,28 +7,34 @@
 from widgetastic_patternfly import Dropdown
 from wrapanapi.containers.image import Image as ApiImage
 
-from cfme.common import SummaryMixin, Taggable, PolicyProfileAssignable
-from cfme.containers.provider import Labelable, navigate_and_get_rows,\
-    ContainerObjectAllBaseView
+from cfme.common import (WidgetasticTaggable, PolicyProfileAssignable,
+                         TagPageView)
+from cfme.containers.provider import (Labelable, navigate_and_get_rows,
+    ContainerObjectAllBaseView, ContainerObjectDetailsBaseView, click_row,
+    LoadDetailsMixin)
 from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import toolbar as tb, CheckboxTable, match_location, InfoBlock,\
-    flash, PagedTable
 from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
 from cfme.utils.appliance import Navigatable
 from cfme.configure import tasks
 from cfme.utils.wait import wait_for, TimedOutError
+from widgetastic_manageiq import SummaryTable
 
-list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
-paged_tbl = PagedTable(table_locator="//div[@id='list_grid']//table")
+
+class ImageAllView(ContainerObjectAllBaseView):
+    SUMMARY_TEXT = "Container Images"
 
 
-match_page = partial(match_location, controller='container_image',
-                     title='Images')
+class ImageDetailsView(ContainerObjectDetailsBaseView):
+    configuration = Dropdown('Configuration')
+    configuration_table = SummaryTable(title='Compliance')
```

Comment:
we put SummaryTable into entities view and Configuration and other menu buttons into toolbar view.
It would be good if you followed same approach.